File: leetcode/javascript/217.js
// Leetcode 217 - Contains Duplicate
// Nov. 25, 2023

/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let visited = new Set();
    for (let i of nums) {
        if (visited.has(i)) return true;
        visited.add(i);
    }
    return false;
};

File: leetcode/python/217.py
## Leetcode 217 - Contains Duplicate
## Nov. 25, 2023

from typing import List

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        visited = set()
        for i in nums:
            if i in visited:
                return True
            visited.add(i)
        return False